submissions/api.py

```diff
@@ -737,13 +737,14 @@ def reset_score(student_id, course_id, item_id, clear_state=False):
 
     # Create a "reset" score
     try:
-        Score.create_reset_score(student_item)
+        score = Score.create_reset_score(student_item)
         # Send a signal out to any listeners who are waiting for scoring events.
         score_reset.send(
             sender=None,
             anonymous_user_id=student_id,
             course_id=course_id,
             item_id=item_id,
+            created_at=score.created_at,
```

Comment:
Since we're interested in the score related to a particular submission, not the submission itself, `created_at` is the value we want.